kube-core/src/resource.rs

```diff
@@ -1,6 +1,5 @@
 pub use k8s_openapi::apimachinery::pkg::apis::meta::v1::ObjectMeta;
 use k8s_openapi::apimachinery::pkg::apis::meta::v1::OwnerReference;
-use once_cell::sync::Lazy;
```

Comment:
I believe `once_cell` is only used here, so it can be removed from `kube-core/Cargo.toml`.